packages/core/src/components/menu/menuDivider.tsx

```diff
@@ -51,7 +51,7 @@ export class MenuDivider extends React.Component<MenuDividerProps> {
         } else {
             // section header with title
             return (
-                <li className={classNames(Classes.MENU_HEADER, className)} role="separator">
+                <li className={classNames(Classes.MENU_HEADER, className)} role="separator" tabIndex={0}>
```

Comment:
```suggestion
                <li className={classNames(Classes.MENU_HEADER, className)} role="separator" tabIndex={-1}>
```

see discussion in https://github.com/palantir/blueprint/issues/6054#issuecomment-1513424391